### PR TITLE
Request to review comments.

### PR DESCRIPTION
**Pull request by TowleT**
Editorial review.

### CODE REVIEW

File: api/html5_deviceMotion.js [hunk 1]
```diff
@@ -99,8 +99,8 @@
  * Acceleration data is given in meters per second squared and uses the body axes defined in the device orientation event. Acceleration data (without the effect of gravity) cannot be provided when there is a lack of a gyroscope. 
  * Implementations that are unable to provide acceleration data without the effect of gravity may instead supply the acceleration including the effect of gravity. 
  * <p>The values reported are equal to the acceleration of the device, plus an acceleration equal and opposite to the acceleration due to gravity.
- * This is less useful in many applications but is provided as a means of providing best-effort support. 
- * These values can be stored and manipulated in variables outside the event scope. These values can be used to control the motion of objects defined on the DOM by the device's gyroscope and accelerometer. 
+ * This is less useful in many applications but is provided as a means of providing best-effort support. What does this mean?
```
Comment (jeffheifetz) [inline api/html5_deviceMotion.js:102]: Remember to remove this

File: api/html5_deviceMotion.js [hunk 2]
```diff
@@ -194,16 +194,16 @@ RotationRate={
 };
 
 /**
- * @toc {Device Orientation} HTML5 Device Motion Event
+ * @toc {Device Orientation} HTML5 Device Motion Event.
```
Comment (jeffheifetz) [inline api/html5_deviceMotion.js:197]: I don't think we want a period on the Table of Contents entries

File: api/html5_deviceMotion.js [hunk 3]
```diff
@@ -194,16 +194,16 @@ RotationRate={
 };
 
 /**
- * @toc {Device Orientation} HTML5 Device Motion Event
+ * @toc {Device Orientation} HTML5 Device Motion Event.
  * @PB10
  * @namespace
- * This event provides direct access to motion data from an accelerometer and gyroscope. The three pieces of data provided are the acceleration, acceleration including gravity and rotation rate. <br>Because our hardware currently does not have a gyroscope, not all interfaces of this event are available to us (acceleration without gravity and rotation rate.)<br>
+ * This event provides direct access to motion data from an accelerometer and gyroscope. The three pieces of data provided are the acceleration, acceleration including gravity and rotation rate. <br>Because the PlayBook currently does not have a gyroscope, not all interfaces of this event are available i.e.(acceleration without gravity and rotation rate.)<br>
```
Comment (jeffheifetz) [inline api/html5_deviceMotion.js:200]: If this is true, why do we have a Rotation Rate class

File: api/html5_deviceMotion.js [hunk 4]
```diff
@@ -219,7 +219,7 @@ DeviceMotionEvent = {
 	
 	/**
 	 * @type Acceleration
-	 * @desc Implementations that are unable to provide acceleration data without the effect of gravity (due, for example, to the lack of a gyroscope) may instead supply the acceleration including the effect of gravity. This is less useful in many applications but is provided as a means of providing best-effort support. 
+	 * @desc Implementations that are unable to provide acceleration data without the effect of gravity (due, for example, to the lack of a gyroscope) may instead supply the acceleration including the effect of gravity. This is less useful in many applications but is provided as a means of providing best-effort support. Not what this means to a developer??
```
Comment (jeffheifetz) [inline api/html5_deviceMotion.js:222]: Remember to remove this

File: api/html5_touch.js [hunk 5]
```diff
@@ -122,9 +122,9 @@ Touch.prototype.radiusY = { };
 
 /**
 *
-* the angle (in degrees) that the ellipse described by radiusX and radiusY is rotated 
+* The angle (in degrees) that the ellipse described by radiusX and radiusY is rotated 
 * clockwise about its center; 0 if no value is known. The value must be greater than 
-* or equal to 0 and less than 90. <br /><br />
+* or equal to 0 and less than 90. Corrected grammar and typos.<br />
```
Comment (jeffheifetz) [inline api/html5_touch.js:127]: Why is this comment here

**Comment by TowleT** [comment]
Reopening to pull in delta branch

**Pull request closed without merging**